app/views/items/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Create a New Merchant Item</h1>
+
+<%= form_with url: merchant_items_path(@merchant), method: :post, local: true do |f| %>
```

Comment:
You could consider refactoring this to use the form_with models like you did on the edit form.  With strong params, that's probably why you had to use `item[name]` instead of just `name`.